mad-dog/src/tests/test_userReplicaSetNodes/stopRandomCreatorNode.js

```diff
@@ -0,0 +1,25 @@
+const _ = require('lodash')
+
+const MadDog = require('../../madDog.js')
+
+/**
+ * Stops a random creator node with maddog
+ * @param {Object} creatorNodeIDToInfoMapping Mapping of creator ndoe ids to info
+ * @returns 
+ */
+const stopRandomCreatorNode = async (creatorNodeIDToInfoMapping) => {
+  const cnIds = Object.keys(creatorNodeIDToInfoMapping).map(id => parseInt(id))
+  const maxId = Math.max(cnIds)
+  const  madDog = new MadDog({ 
+    numCreatorNodes: maxId - 1,
+    downProbability: 0,
```

Comment:
why downProbability of 0?

Reply:
`downProbability` in the MadDog class means the chances that MadDog pauses the docker container from running. This is not used in this test because we just disable networking to that node. Can change to make more clear.

Reply:
ahh ok yeah that was confusing, thanks